### PR TITLE
v1.3.0

### PR DESCRIPTION
**Pull request by bartlomieju**
<s>Waiting on https://github.com/denoland/deno/pull/6940</s>
<s>Waiting on https://github.com/denoland/deno/pull/7040</s>

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ https://github.com/denoland/deno/releases
...
+Changes in the CLI:
+
+- feat: Add "--no-check" flag to deno install (#6948)
+- feat: Add "--ignore" flag to deno lint (#6934)
+- feat: Add "--reload" flag to deno bundle (#6996)
+- feat: Add "--reload" flag to deno info (#7009)
+- feat: Stabilize Deno.mainModule (#6993)
+- feat: Support file URLs in Deno.run for executable (#6994)
+- feat(doc): handle imports (#6987)
+- feat(web): FileReader API (#6673)
```
Comment (ry) [inline Releases.md:20]: (web) designation isn't so useful... suggest removing

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ https://github.com/denoland/deno/releases
...
+
+Changes in the CLI:
+
+- feat: Add "--no-check" flag to deno install (#6948)
+- feat: Add "--ignore" flag to deno lint (#6934)
+- feat: Add "--reload" flag to deno bundle (#6996)
+- feat: Add "--reload" flag to deno info (#7009)
+- feat: Stabilize Deno.mainModule (#6993)
+- feat: Support file URLs in Deno.run for executable (#6994)
+- feat(doc): handle imports (#6987)
```
Comment (ry) [inline Releases.md:19]: prefer `feat: handle imports in deno doc`

File: op_crates/web/Cargo.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 [package]
 name = "deno_web"
-version = "0.2.0"
+version = "0.3.0"
```
Comment (bartlomieju) [inline op_crates/web/Cargo.toml:5]: @ry actually not sure if we need to release new version of `deno_web`, and... does it deploy automatically?

Comment (ry) [reply op_crates/web/Cargo.toml:5]: It does. Let's just cut a new release even tho there are no changes. If not the crate publishing mechanism may break.

Comment (bartlomieju) [reply op_crates/web/Cargo.toml:5]: Great 👍

**Pull request merged**